### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by chris-gds**
## What 

Add suggested updates from Content about how to contribute
Combining suggested updates with forking details.

```
##Contribute

This WCAG Primer is for everyone.You can help make sure it stays up to date by: 

making changes to the [WCAG primer on Github](https://github.com/alphagov/wcag-primer) 
emailing the Accessibility Capability team at <accessibility@digital.cabinet-office.gov.uk> with suggestions  
```

## Why?

Part of on-going a11y work to aggregate, update and surface accessibility resources.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,13 @@ https://alphagov.github.io/wcag-primer/
...
 # Contributing
 
-To contribute to this repository, you first need to [fork it](https://help.github.com/en/articles/fork-a-repo). You can raise PRs from your forked copy.
+This WCAG Primer is for everyone.You can help make sure it stays up to date by:
+
+1. Making changes to the [WCAG primer on Github](https://github.com/alphagov/wcag-primer).
+
+To contribute to this repository, you first need to [fork it](https://help.github.com/en/articles/fork-a-repo)* You can raise PRs from your forked copy.
+
+2. Emailing the accessibility capacity team at <accessibility@digital.cabinet-office.gov.uk> with suggestions
```
Comment (maxgds) [inline README.md:21]: We've got the team name wrong, it's the Accessibility Capability team (probably wants capitals though check with a content designer).

**Review by maxgds — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chris-gds) [reply README.md]: I've updated with correction and requested confirmation from Content

File: README.md [hunk 2]
```diff
@@ -12,7 +12,13 @@ https://alphagov.github.io/wcag-primer/
 
 # Contributing
 
-To contribute to this repository, you first need to [fork it](https://help.github.com/en/articles/fork-a-repo). You can raise PRs from your forked copy.
+This WCAG Primer is for everyone.You can help make sure it stays up to date by:
```
Comment (maxgds) [inline README.md:15]: Please add a space after the full stop.

**Comment by chris-gds** [comment]
[Closing as raised here ](https://github.com/alphagov/wcag-primer/pull/76) (due to default br rename)

**Pull request merged**